packages/grafana-data/src/field/displayProcessor.test.ts

```diff
@@ -305,7 +305,7 @@ describe('Format value', () => {
     const value = 1200;
     const instance = getDisplayProcessorFromConfig({ decimals: null, unit: 'short' });
     const disp = instance(value);
-    expect(disp.text).toEqual('1.2');
+    expect(disp.text).toEqual('1.20');
```

Comment:
is this reverting the change in #57373?

are there any tests that show/enforce what the new behavior is?